lib/src/main/kotlin/at/bitfire/ical4android/AndroidCompatTimeZoneRegistry.kt

```diff
@@ -10,6 +10,11 @@ import net.fortuna.ical4j.model.component.VTimeZone
 import net.fortuna.ical4j.model.property.TzId
 import java.time.ZoneId
 import java.util.logging.Logger
+import net.fortuna.ical4j.model.Property
+import net.fortuna.ical4j.model.component.Daylight
+import net.fortuna.ical4j.model.component.Standard
+import net.fortuna.ical4j.model.property.TzUrl
+import net.fortuna.ical4j.model.property.XProperty
```

Comment:
Do we still need these imports?

Reply:
Solved, I thought I had automatic import optimization enabled but I didn't. I've fixed it now